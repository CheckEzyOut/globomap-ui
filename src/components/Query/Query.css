.queries {
  padding: 0 25px;
  background-color: none;
}

.queries ul {
  list-style: none;
  padding: 0;
}

.queries .query {
  display: block;
  margin-bottom: 10px;
  height: 30px;
  line-height: 30px;
  text-decoration: none;
  background: none;
  border-radius: 3px;
  color: #333;
}

.queries .query:before {
  content: "Â»";
  margin-right: 10px;
}

.queries .query:hover {
  color: #2eb78e;
}
