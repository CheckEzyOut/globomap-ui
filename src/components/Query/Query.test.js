import React from 'react';
import { shallow } from 'enzyme';
import { Query } from './Query';

describe("Query", function() {

  it('should render Query component', () => {
    const component = shallow(<Query />);
    expect(component).toMatchSnapshot();
  });

});
