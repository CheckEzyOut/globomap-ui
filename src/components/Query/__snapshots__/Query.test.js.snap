// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Query should render Query component 1`] = `""`;
